src/ui/atoms/listItem/ListItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import type { DeepReadonly } from 'superTypes'
+import { Typography } from '../typography/Typography'
+import './listItem.scss'
+import { isString } from 'utils'
+
+export type ListItemProps = {
+  /**
+   * The main label of the item.
+   */
+  readonly label: string
+  /**
+   * Additional information about the item
+   */
+  readonly description?: string | JSX.Element
+  /**
+   * An icon that can provide visual information on the left side.
+   */
+  readonly leftIcon?: JSX.Element
+  /**
+   * An icon that can provide visual information on the right side.
+   */
+  readonly rightIcon?: JSX.Element
```

Comment:
As I said above, I am not sure about those icon's properties, for many reasons:
- it could be something other than icons (text for example, a button, etc.)
- in my opinion, it should be part of the description itself (or the main content of the `ListItem`)
- the style is imposed, maybe that should be the responsibility of the caller? (it's an open question)